socorro/unittest/external/postgresql/unittestbase.py

```diff
@@ -119,6 +127,14 @@ def setUpClass(cls):
         case (aka. test class).
         """
         cls.config = cls.get_standard_config()
+        if cls.config.get('database_url'):
+            url = urlparse.urlparse(cls.config.database_url)
+            cls.config.database_username = url.username
+            cls.config.database_password = url.password
+            cls.config.database_hostname = url.hostname
+            cls.config.database_port = url.port
+            cls.config.database_name = url.path[1:]
+
         cls.database = db.Database(cls.config)
```

Comment:
Shouldn't this be solved inside that `db.Database` class instead? I.e. *it* could look out for a `database_url` key.